lms/djangoapps/courseware/tabs.py

```diff
@@ -66,8 +65,7 @@ def is_enabled(cls, course, user=None):
         """
         The "Home" tab is not shown for the new unified course experience.
         """
-        request = RequestCache.get_current_request()
-        return not waffle.flag_is_active(request, UNIFIED_COURSE_EXPERIENCE_FLAG)
+        return not UNIFIED_COURSE_EXPERIENCE_FLAG.is_enabled(course.id)
```

Comment:
I'm wondering if this should be UNIFIED_COURSE_TABS_FLAG, since we might have other flags for the full experience. I named this in the first place, but now that we have a namespace it seems too general a name. What do you think?